touchforms/formplayer/experiments.py

```diff
@@ -24,6 +24,9 @@ def publish(self, result):
             self.log_diff(control, candidate)
             return
 
+        if candidate.failure:
+            logging.info("Candidate %s failed with exception %s" % (candidate, candidate.exception))
```

Comment:
this isn't using one of the loggers you defined at the top of the file (and there are some other places in here that aren't either). is that intentional?

Reply:
Yes - I don't want to log to the diff file in this case, I just want to see the exception that occurred in the machine logs